src/views/ConnectWallet.vue

```diff
@@ -0,0 +1,129 @@
+<template>
+  <div>
+    <div class="p-sm-2">
+      <div class="my-3">
+        <div>
+          <h1>connect wallet</h1>
+          <p>Connect your Bitcoin or Lightning wallet to your Umbrel</p>
+        </div>
+      </div>
+
+      <b-row>
+        <b-col cols="12" md="4" xl="3">
+          <b-form-select
+            :value="wallet"
+            :options="options"
+            @change="selectWallet"
+            class="mb-4"
+          ></b-form-select>
+        </b-col>
+      </b-row>
+
+      <router-view :urls="urls" @showQrModal="showQrModal"></router-view>
+    </div>
+
+    <b-modal
```

Comment:
This modal needs padding for narrower displays.

<img width="796" alt="Screenshot 2021-02-14 at 20 20 03" src="https://user-images.githubusercontent.com/2123375/107878052-c4f5e880-6f02-11eb-9647-7ef2324b322d.png">

Reply:
Is it still like that if you refresh the page and then open the modal? Because it doesn't dynamically adjust the size after first load.

Reply:
Yep, happens without resizing after page load.